[2020/08/03 16:03:35.870854]========== Start ==========
[2020/08/03 16:03:35.874865][I][main] Python 3.7.2 (tags/v3.7.2:9a3ffc0492, Dec 23 2018, 23:09:28) [MSC v.1916 64 bit (AMD64)]
[2020/08/03 16:03:35.876870][I][main] GFI20LastScript.py 3.0.0.6
[2020/08/03 16:03:35.879916][I][readINI] ---------- INI ----------
[2020/08/03 16:03:35.883497][I][readINI] StartButton_ON = False
[2020/08/03 16:03:35.885956][I][readINI] PowerSupplier = 66321B
[2020/08/03 16:03:35.887006][I][readINI] PowerSupplierAddr = GPIB0::5::INSTR
[2020/08/03 16:03:35.889793][I][readINI] DigitalMeter = 8261A
[2020/08/03 16:03:35.890840][I][readINI] DigitalMeter_COMPort = 1
[2020/08/03 16:03:35.893818][I][readINI] ELoadTestState = False
[2020/08/03 16:03:35.900464][I][readINI] Voltage_Ch = 107
[2020/08/03 16:03:35.903518][I][readINI] Voltage_Max = 0.3
[2020/08/03 16:03:35.906588][I][readINI] Voltage_Min = 0
[2020/08/03 16:03:35.908803][I][readINI] DetectVoltage = 0.3
[2020/08/03 16:03:35.910810][I][readINI] TestTimeout = 50.0
[2020/08/03 16:03:35.914823][I][readINI] LeastTestTime = 2.5
[2020/08/03 16:03:35.918350][I][readINI] EC = 010000S03
[2020/08/03 16:03:35.920362][I][readINI] ---------- INI ----------
[2020/08/03 16:03:35.922271]---------- startTest ----------
[2020/08/03 16:03:35.925323]================ Control DM ================
[2020/08/03 16:03:35.929326]==================Setup digital meter==================
[2020/08/03 16:03:35.942252][I][setupDigitalMeter] Get visa resource manager...
[2020/08/03 16:03:35.966353][I][setupDigitalMeter] Get visa resource instrument...
[2020/08/03 16:03:35.974998][E][setupDigitalMeter] Connect to digital meter Error
[2020/08/03 16:03:35.979995][E][startTest] Control digital meter failed!
[2020/08/03 16:03:35.982856][I][TestResult] Test Fail! Control digital meter failed!
